analyzers/ngram_stats/interface.py

```diff
@@ -27,6 +31,56 @@
         OutputColumn(name=COL_NGRAM_TOTAL_REPS, data_type="integer"),
         OutputColumn(name=COL_NGRAM_DISTINCT_POSTER_COUNT, data_type="integer")
       ]
+    ),
+    AnalyzerOutput(
+      id=OUTPUT_NGRAM_FULL,
+      name="N-gram full report",
+      columns=[
+        OutputColumn(
+          name=COL_NGRAM_ID, data_type="identifier",
+          human_readable_name="ngram ID"
+        ),
+        OutputColumn(
+          name=COL_NGRAM_LENGTH, data_type="integer",
+          human_readable_name="ngram length"
+        ),
+        OutputColumn(
+          name=COL_NGRAM_WORDS, data_type="text",
+          human_readable_name="ngram content"
+        ),
+        OutputColumn(
+          name=COL_NGRAM_TOTAL_REPS, data_type="integer",
+          human_readable_name="ngram frequency"
+        ),
+        OutputColumn(
+          name=COL_NGRAM_DISTINCT_POSTER_COUNT, data_type="integer",
+          human_readable_name="distinct user count"
+        ),
+        OutputColumn(
+          name=COL_AUTHOR_ID, data_type="identifier",
+          human_readable_name="unique username"
+        ),
+        OutputColumn(
+          name=COL_MESSAGE_NGRAM_COUNT, data_type="integer",
```

Comment:
minor: given the human readable description, would this constant name be more meaningful as `COL_MESSAGE_NGRAM_COUNT_BY_USER`. Otherwise, it seems to suggest similar meaning to `COL_NGRAM_TOTAL_REPS`. 

Just a suggestion.

Reply:
Good suggestion, let me change that. Thanks!

Reply:
Actually, good catch, this is misusing a different column symbol that's meant for the number of repetitions of an ngram within a _single_ message, when really a new column symbol is needed, one that denotes total number of repetitions across all messages for a single user. Thanks!